### ISSUE TITLE
Search Youtube videos via link?

### ISSUE THREAD

**Issue by markusd1984** [mattwright324/youtube-comment-suite#22]
It would make it easier to be able to search via youtube link, if that where possible?

As per https://github.com/mattwright324/youtube-comment-suite/issues/4

In the drop down under Relevance, Date, Title, Rating, Views to include one with "Link". :) 

P.S. Sometimes even copy/paste of the entire title is not finding the exact video match, probably due to special characters etc.

**Comment by mattwright324** [comment]
This sounds like a good feature and maybe a bug to fix too!

Would you mind sharing the video(s) at fault with me so that I could see why searching it's title isn't working?

**Comment by markusd1984** [comment]
Great, the link would be so convenient and I would expect most people will have one.

I think they contained a question mark or double point in the title.

**Comment by markusd1984** [comment]
I only now realized, that I didn't needed to search the video first but instead I can simply use the "add Items" within "Manage Group" which expects YT links 😄 .

Tbh. just looking at the app at first time, the work flow to me wasn't as obvious or intuitive, especially as the first tab is "searching comments", which requires first for videos to be added to the default group or a newly created group.

In other words, there is no quick way to get from a video straight into the comment search 😃 .
(Sure I can see for opening the app again and again it's more convenient to have the comments search open.)

**Comment by mattwright324** [comment]
I've added specific video searching using the video id as shown below. I'll have it out in the next release.

![](https://i.imgur.com/HAVtCTl.png)

Not exactly the original purpose of this issue so I can close this, I'll keep that workflow stuff in mind.
